src/main/java/com/auth0/json/mgmt/EmailVerificationIdentity.java

```diff
@@ -0,0 +1,52 @@
+package com.auth0.json.mgmt;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+
+/**
+ * Represents the identity object that can be sent on requests to create an email verification ticket or job.
+ * Related to {@link com.auth0.client.mgmt.JobsEntity} and {@link com.auth0.client.mgmt.TicketsEntity}.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class EmailVerificationIdentity implements Serializable {
```

Comment:
is the serializable really needed?

Reply:
Good catch. No, I don't think it is, will confirm. Looks like we have a few models that do implement Serializable for some reason.

Reply:
Updated to not be Serializable.